Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM node:14-buster-slim
+
+WORKDIR /app
+
+COPY package.json package-lock.json ./
+
+RUN npm install --production
+
+COPY layouts layouts
+COPY lib lib
+COPY views views
+COPY sg/db.json sg/
+
+# TODO
+COPY config.json config.json
```

Comment:
`config.json` is fine to copy and doesn't need customization, so you can take it out of the TODO block :).